libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -872,22 +905,22 @@ method unsubscribePeer*(g: GossipSub, peer: PeerID) =
     # also try to remove from explicit table here
     g.explicit.removePeer(t, pubSubPeer)
 
-    when defined(libp2p_expensive_metrics):
+    if KnownLibP2PTopicsSeq.contains(t):
```

Comment:
let's catch all other subscriptions in `other` or something, so we can track how many topics get subscribed - ie this would allow us to track topic oversubscription

Reply:
I could not find an acceptable way and so decided to skip this.

So what you propose is to anyway increase the counter summing up all other topics? in that case we could make observations based on averages etc? ( A bit like my dirty current metric on master )

Reply:
well, the point would be to detect spikes that can be correlated with other issues

Reply:
Sounds good